hockeysdk/build.gradle

```diff
@@ -52,7 +52,7 @@ publish {
 }
 
 dependencies {
-    compile fileTree(dir: 'libs', include: ['*.jar'])
+    compile "com.android.support:support-annotations:$supportLibVersion"
```

Comment:
Just a small thing to note here: Moving this dependency from `androidTestCompile` to `compile` will *add* a transitive dependency to `net.hockeyapp.android:HockeySDK`. Previously the library had *zero* dependencies. While I think this is totally fine (especially since `support-annotations` is such a widely used, lightweight, and useful library) - it might be advisable to not add it as a "side-effect" in a commit called `Clean up code from warnings, remove unused utils`. Here, it is only used in two classes. Instead, it could be moved to a separate commit and be integrated in many more places.